flow/activities/flowable_core.go

```diff
@@ -60,6 +60,7 @@ func syncCore[TPull connectors.CDCPullConnectorCore, TSync connectors.CDCSyncCon
 	config *protos.FlowConnectionConfigs,
 	options *protos.SyncFlowOptions,
 	sessionID string,
+	adaptStream func(*model.CDCStream[Items]) (*model.CDCStream[Items], error),
```

Comment:
make `adaptStream` nil in case of PgValue or nil script instead of adding the check within `adaptStream`

Reply:
go doesn't support nil function values